libs/shared/console-shared/src/lib/application-settings/ui/application-settings-resources/application-settings-resources.tsx

```diff
@@ -28,8 +31,11 @@ export interface ApplicationSettingsResourcesProps {
 export function ApplicationSettingsResources(props: ApplicationSettingsResourcesProps) {
   const { displayWarningCpu, application, minInstances = 1, maxInstances = 50, clusterId = '', environmentMode } = props
   const { control, watch } = useFormContext()
-  const { organizationId = '', environmentId = '', applicationId = '' } = useParams()
+  const { organizationId = '', projectId = '', environmentId = '', applicationId = '' } = useParams()
   const { data: runningStatuses } = useRunningStatus({ environmentId, serviceId: applicationId })
+  const { data: environment } = useFetchEnvironment(projectId, environmentId)
```

Comment:
use `useEnvironment` from `@qovery/domains/environments/feature'` instead of `useFetchEnvironment`